code/web/src/modules/user/Profile.js

```diff
@@ -31,16 +35,42 @@ const Profile = (props) => (
     </Grid>
 
     <Grid>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        {/*<Tile style={{}} image={icon}></Tile>*/}
+        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
+      </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
+        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
+        {/*Description text should be dynamic from user details ie props.user.details.description */}
+      </GridCell>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
-
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
-
+        <p style={{ color: grey2, marginBottom: '2em' }}>123 West 5th St City, State 80654</p>
+        {/*Address text should be dynamic from user details ie props.user.details.shipping */}
```

Comment:
@fieldstyler @TravisGM92 @EHale64 , I am curious how ya'll are deciding to handle specifically the shipping adress?
We could use a single string, or an object with key values for number, street, city, state, zip.
This isn't too time sensitive but the sooner we know the better.